public/admin.php

```diff
@@ -87,7 +87,14 @@
                     $message .= "<br>";
                 }
                 recalculatePlayerPoints($validUser);
-                $message .= "- Recalculated Score: " . getPlayerPoints($validUser) . "<br>";
+
+                $hardcorePoints = 0;
+                $softcorePoints = 0;
+                if (getPlayerPoints($awardAchievementUser, $userPoints)) {
```

Comment:
I think `$awardAchievementUser` is a CSV (see preg_split on line 68). It's probably best to just use `$validUser` as that's whose points were recalculated on line 89.